lib/logstash/outputs/elasticsearch/elasticsearch-template-es5x.json

```diff
@@ -9,7 +9,7 @@
       "_all" : {"enabled" : true, "norms" : false},
       "dynamic_templates" : [ {
         "message_field" : {
-          "match" : "message",
+          "path_match" : "message",
           "match_mapping_type" : "string",
           "mapping" : {
             "type" : "string", "index" : "analyzed", "norms" : false,
```

Comment:
re: your first comment wrt `text` vs `string`. I will change it.

from the documentation, I was kind of shocked this didn't fail https://www.elastic.co/guide/en/elasticsearch/reference/5.0/string.html.

After looking at the code, looks like ES optimistically upgrades this into a `text` mapping since we also provide `"index": "analyzed"`. Still, I can update